Source/Core/Core/HW/DSPHLE/UCodes/Zelda.cpp

```diff
@@ -1536,12 +1537,12 @@ void ZeldaAudioRenderer::Resample(VPB* vpb, const s16* src, MixingBuffer* dst)
   vpb->current_pos_frac = pos & 0xFFF;
 }
 
-void* ZeldaAudioRenderer::GetARAMPtr() const
+void* ZeldaAudioRenderer::GetARAMPtr(u32 offset) const
 {
-  if (m_aram_base_addr)
-    return HLEMemory_Get_Pointer(m_aram_base_addr);
+  if (SConfig::GetInstance().bWii)
```

Comment:
Not worth blocking over IMO, but I've heard mention of deferring SConfig to instead use the new config

Reply:
`bWii` isn't something that would make sense to move to the new config system since it's not an actual setting, more of a global variable. It probably doesn't make sense to have it in `SConfig`, but moving it somewhere else would be part of a larger refactor (which is definitely out of scope for this PR).

Reply:
Agreed. Adding new settings to SConfig would be rejected, but it's fine to keep using whatever is in there for the time being.